telegram_export/utils.py

```diff
@@ -211,3 +213,40 @@ def action_to_name(action):
         types.ChannelAdminLogEventActionToggleSignatures: 'toggle.signatures',
         types.ChannelAdminLogEventActionUpdatePinned: 'update.pinned',
     }.get(type(action), None)
+
+
+def parse_proxy_str(proxy_str):
```

Comment:
…in this method you need `if not socks: raise`, advicing the user to install the dependency if they want to use a proxy.